.github/ranger.yml

```diff
@@ -23,3 +25,7 @@ labels:
     action: comment
     delay: 5s
     message: "Thanks for making your first contribution! :slightly_smiling_face:"
+  extension-request:
+    action: close-faster
+    delay: 5s
+    message: "Thanks for opening an extension request! We are currently in the process of switching extension marketplaces and transitioning over to [Open VSX](https://open-vsx.org/). Once https://github.com/eclipse/openvsx/issues/249 is implemented, we can fully make this transition. Therefore, we are no longer accepting new requests for extension requests. We suggest installing the VSIX file and then installing into code-server as a temporary workaround. See [docs](https://github.com/cdr/code-server/blob/main/docs/FAQ.md#installing-vsix-extensions-via-the-command-line) for more info."
```

Comment:
Is there a way to wrap this? And does the link look right?

Reply:
Yes! You can do long lines in YAML using `|` or `>` https://en.wikipedia.org/wiki/YAML#Basic_components

Try:

```yaml
extension-request:
  message: >
    Thanks for opening an extension request!
    We are currently in the process of
```

This is if you want it to be a single paragraph, you can also use `|` to preserve the newlines if you want (which of course might not make a difference if this message is Markdown)

Reply:
TIL!
You're the man! Thanks @jawnsy